conf.py

```diff
@@ -56,11 +56,6 @@
     "logo": "logo.png"
 }
 
-# Global substitutions
-rst_prolog = """
-.. |m4| replace:: March IV
-"""
```

Comment:
I wouldn't remove the global substitution, but give it a more general name instead